README.md

```diff
@@ -76,7 +76,8 @@ Before contributing a change to this repository, please first discuss the change
 4. Run `npm run build` which generates a `lib` directory with the transpiled `es5` code.
 5. Fire in `npm link` to test everything works fine.
 6. Commit and push your changes.
-7. Make a detailed pull request.
+7. Run e2e tests in github actions ci pipeline within your own fork. All tests should pass.
```

Comment:
I think it is better to ask them to execute tests locally (`npm test`) before making a commit.

Reply:
ok will update